src/PageContent/Calendar/DelivererGroupContent.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import { AccountType, StringFormat } from '../../Enums';
+import './Content.css';
+import volunteer from '../../icons/volunteer.svg';
+
+class DelivererGroupContent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            edit: false,
+            delivererGroup: this.props.delivererGroup,
+            deliverer1: this.props.delivery.delivererGroup.deliverers[0].name,
+            deliverer2: this.props.delivery.delivererGroup.deliverers[1].name,
+            phone1: this.props.delivery.delivererGroup.deliverers[0].phone,
+            phone2: this.props.delivery.delivererGroup.deliverers[1].phone,
+            email1: this.props.delivery.delivererGroup.deliverers[0].email,
+            email2: this.props.delivery.delivererGroup.deliverers[1].email
```

Comment:
the `deliverers` list could also be empty if no deliverers have been assigned yet. So instead of expanding the data out, you can probably keep the structure the same as the dummy data for easier assigns and allow checks in render. Something like:
```
this.state = {
    deliverers: this.props.delivery.delivererGroup.deliverers
}
```